src/controllers/Enterprise/putEnterpriseController.js

```diff
@@ -1,18 +1,18 @@
 const { Enterprise } = require("../../database");
 
 const putEnterpriseController = async (ruc, newRuc, name, address) => {
-    const enterprise = await Enterprise.findOne({ where: { ruc } });
+  const enterprise = await Enterprise.findOne({ where: { ruc } });
 
-    newRuc ? (enterprise.ruc = newRuc) : null;
-    name ? (enterprise.name = name) : null;
-    address ? (enterprise.address = address) : null;
+  newRuc ? (enterprise.ruc = newRuc) : null;
+  name ? (enterprise.name = name) : null;
+  address ? (enterprise.address = address) : null;
 
-    await enterprise.save();
+  await enterprise.save();
 
-    return {
-        updated: true,
-        ruc,
-    };
+  return {
+    updated: true,
```

Comment:
Same here.

Reply:
Ok. I'll resolve the conversation once the task is finished.